http.kafka.sasl.scram/stack.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+networks:
+  net0:
+    driver: overlay
+
+services:
+
+  zilla:
+    image: "ghcr.io/aklivity/zilla:develop-SNAPSHOT"
```

Comment:
Please update to `latest` docker image. 😄

Reply:
done